apps/hip/wagtail_hooks.py

```diff
@@ -0,0 +1,12 @@
+from django.templatetags.static import static
+from django.utils.html import format_html
+
+from wagtail.core import hooks
+
+
+@hooks.register("insert_editor_css")
+def editor_css():
+    """Register the extra CSS styles to be used for the Wagtail editor."""
+    return format_html(
+        '<link rel="stylesheet" href="{}">', static("styles/wagtail_styles.css")
```

Comment:
Wonder if we could use `sass_processor()` and then wagtail_styles could be a scss file instead of css:
https://github.com/jrief/django-sass-processor#in-media-classes-or-properties